java_grpc_library.bzl

```diff
@@ -109,14 +109,18 @@ def java_grpc_library(name, srcs, deps, flavor=None,
   added_deps = [
       "@grpc_java//core",
       "@grpc_java//stub",
-      "@grpc_java//protobuf",
       "@com_google_guava_guava//jar",
   ]
   if flavor == "normal":
-    added_deps += ["@com_google_protobuf//:protobuf_java"]
+    added_deps += [
+        "@com_google_protobuf//:protobuf_java",
+        "@grpc_java//protobuf",
```

Comment:
Here is already removed (should not depend on workspace, be like a `//protobuf`) and workspace was renamed to `io_grpc_grpc_java`, please check your branch is up to date.
https://github.com/grpc/grpc-java/blob/ae1fb9467cf65ddb8830f381c073ca1597a8a894/WORKSPACE#L1

Reply:
Thanks! I forgot to rebase on master. Now I have to fix my build first because using master broke it somehow. I'll send an update tomorrow.